content/applications/hr/recruitment/recruitment-flow.rst

```diff
@@ -0,0 +1,233 @@
+a job position, by default.
+
+.. note::
+   If the website application form is modified, different fields may be populated, based on what
+   information is requested on the website.
+
+If the applicant entered any information in the *Short Introduction* section of the online
+application, it populates the :guilabel:`Application Summary` tab at the bottom of the applicant's
+card.
+
+Resumé
+------
+
+If a resumé was attached to the online application, it appears in the :guilabel:`Files` section of
+the chatter, and is also stored in the *Documents* application.
+
+To find the recruitment documents, navigate to the main :menuselection:`Documents app` dashboard,
+and click the :guilabel:`Recruitment` folder on the left-hand side. All recruitment documents are
+stored within that folder.
+
+If the :ref:`CV Display <recruitment/cv-display>` option was enabled in the :ref:`Settings
+<recruitment/settings>` of the *Recruitment* app, the resumé appears on the applicant's card, on the
+right-hand side.
+
+.. note::
+   Depending on the browser zoom level, or size of the browser screen, the resumé may appear below
+   the main applicant card information as a PDF link.
+
+Send interview
+--------------
+
+At any point in the hiring process, an interview can be sent to the applicant to obtain more
+information. These interviews are custom-made, and can be formatted in a variety of ways.
+
+The *Surveys* application is **required** to send interviews to an applicant, so it **must** be
+installed.
+
+Odoo uses the term *interview*, but these can be thought of as questionnaires, surveys, tests,
+certifications, etc. Custom interviews can be formatted to suit each individual job position's
+needs. For more information on creating and editing interviews, refer to the
+:doc:`../../hr/recruitment/new_job` documentation.
+
+.. example::
+   A job position for a computer programmer could have an interview in the form of a programming
+   quiz to determine the skill level of the applicant. A job position for a restaurant server could
+   have a questionnaire inquiring about the applicant's availability, if the desired applicant needs
+   to be available on weekend evenings.
+
+To send an interview to an applicant, first click the applicant's card from the
+:guilabel:`Applications` page, to view the detailed applicant information. At the top-left of the
+applicant's card, click the :guilabel:`Send Interview` button.
+
+If the applicant's card has an email address on file, a :guilabel:`Send an interview` pop-up window
+appears, with the :guilabel:`Recipients`, :guilabel:`Subject`, and email body populated.
+
+.. note::
+   To send an email to an applicant, there **must** be an :guilabel:`Email` address on the
+   applicant's card.
+
+   If an email address is not entered on the applicant's card, when the :guilabel:`Send Interview`
+   button is clicked, an :guilabel:`Edit:(Applicant's Name)` pop-up window appears, *on top of* the
```

Comment:
nitpick 😛 
this gui for **Edit:(Applicant's Name)** does *not* have a space after the colon, however there *is* a space in the same gui on line 92